src/ClaimToken.sol

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
I think we could use the exact `Dai` version instead of the older one. So both tokens have a matching code. The only exception might be the wards access for burning, that we should still discuss if we want it or just make the user to give an approval.

Reply:
Ah yes that pesky burn ward. That's an accident. I'll remove it.

Reply:
Done.